modules/tinymce/src/core/main/ts/api/file/BlobCache.ts

```diff
@@ -90,30 +91,25 @@ export const BlobCache = function (): BlobCache {
     };
   };
 
-  const add = function (blobInfo: BlobInfo) {
+  const add = (blobInfo: BlobInfo) => {
     if (!get(blobInfo.id())) {
       cache.push(blobInfo);
     }
   };
 
-  const get = function (id: string): BlobInfo {
-    return findFirst(function (cachedBlobInfo) {
-      return cachedBlobInfo.id() === id;
-    });
-  };
+  const findFirst = (predicate: (blobInfo: BlobInfo) => boolean) => Arr.find(cache, predicate).getOrUndefined();
```

Comment:
Changed this from `Arr.filter` to `Arr.find` so we don't have to iterate over all cached blobs. I used `getOrUndefined()` to keep the return types the same when no match was found.